geoips/plugins/modules/coverage_checkers/center_radius.py

```diff
@@ -110,21 +108,13 @@ def call(
         Percent coverage of variable_name
     """
     varname_for_covg = variable_name
-    if variable_name not in xarray_obj.variables.keys() and alt_varname is not None:
+    if variable_name not in xarray_obj.variables.keys():
         LOG.info(
-            '    UPDATING variable "%s" does not exist, using alternate "%s"',
+            '    UPDATING variable "%s" does not exist, using product name "%s"',
             variable_name,
-            alt_varname,
+            name,
```

Comment:
Looking at this closer, I don't think this is what we want - it is using the name of the coverage checker plugin if the variable name does not exist in the xarray object.  I don't think that makes sense.  I will think about how this works, and expect an updated implementation when 1.11.1 is released.